wrappers/Dockerfile.xenial

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:xenial
```

Comment:
It is unused, but I wanted to keep it in case we need it again - it was a pain to set up.